src/open-api.ts

```diff
@@ -55,6 +63,7 @@ export class OpenAPI extends OpenApiBuilder {
         // @todo involve config/inputSources in v4
         operation.requestBody = depictRequest(commonParams);
       }
+      path = convertColonToBrackets(path);
       this.addPath(path, {
```

Comment:
I'm not a big fan of mutable stuff, so I try to avoid it where possible.
Luckily, here it's easy:

```typescript
const swaggerSafePath = convertColonToBrackets(path);
this.addPath(swaggerSafePath, {
  ...(this.rootDoc.paths?.[swaggerSafePath] || {}),
  [method]: operation,
});
```